PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -996,7 +1134,8 @@ public void SetOption(PowerShellRawHostOptions options)
         }
 
         /// <summary>
-        /// Returns the connection info for the current runspace
+        /// Returns the connection info for the current runspace based on certain characteristics of the runspace. Note: the criteria for remote debug vs remote session
+        /// is not always correct and should be treated carefully.
         /// </summary>
         public DebugScenario GetDebugScenario()

```

Comment:
This does not seem to be a public service we want to expose? Just utilities function?